src/software/sensor_fusion/sensor_fusion_test.cpp

```diff
@@ -21,7 +21,7 @@ class SensorFusionTest : public ::testing::Test
           geom_data(initSSLDivBGeomData()),
           robot_status_msg_id_1(initRobotStatusId1()),
           robot_status_msg_id_2(initRobotStatusId2()),
-          robot_status_msg_low_cap(initLowCapErrorCode()),
+          robot_status_msg_high_cap(initLowCapErrorCode()),
```

Comment:
rename initLowCapErrorCode -> initHighCapErrorCode

Reply:
CI seems to be failing due to this